### DISCUSSION TITLE
Failed to connect to localhost/127.0.0.1:5001

### DISCUSSION THREAD

**Discussion by IliyanAng** [dotnet/maui#16865] [Show and tell]
This is also a reference to issue: 
https://github.com/dotnet/maui/issues/6705 

Adding a solution here, as the previous issue report is locked. 
When using Emulator to connect to localhost API running instance you can now take advantage of Visual Studio 2022 17.7+ dev tunnels and setup a publicly accessible tunnel from your local device. Copy and paste that url instead of the localhost and you should be all set.

**Comment by matmork** [comment]
Try 10.0.2.2:5001

https://developer.android.com/studio/run/emulator-networking#networkaddresses
